vroom/autopep8.vroom

```diff
@@ -0,0 +1,115 @@
+The built-in autopep8 formatter knows how to format python code.
+
+We'll set up codefmt and configure the vroom environment, then jump into some
+examples. If you haven't seen the setup code below yet, read through main.vroom
+briefly.
+
+  :set nocompatible
+  :let g:repo = fnamemodify($VROOMFILE, ':p:h:h')
+  :execute 'source' g:repo . '/bootstrap.vim'
+  :call maktaba#syscall#SetUsableShellRegex('\v<shell\.vroomfaker$')
+
+  :let g:repeat_calls = []
+  :function FakeRepeat(...)<CR>
+  |  call add(g:repeat_calls, a:000)<CR>
+  :endfunction
+  :call maktaba#test#Override('repeat#set', 'FakeRepeat')
+
+  :call codefmt#DisableIsAvailableChecksForTesting()
+
+
+The autopep8 formatter expects the autopep8 executable to be installed on your
+system.
+
+  :FormatCode autopep8
+  ! autopep8 .*
+  $ 
+
+The name or path of the autopep8 executable can be configured via the
+autopep8_executable flag if the default of "autopep8" doesn't work.
+
+  :Glaive codefmt autopep8_executable='autopep20'
+  :FormatCode autopep8
+  ! autopep20 .*
+  $ 
+  :Glaive codefmt autopep8_executable='autopep8'
```

Comment:
Are these glaive commands clearing the cached --version?  Should we use that everywhere?